aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -1,129 +1,107 @@
-                params = self._convert_params(params)
-
-                # Handle OmegaConf object
-                try:
-                    from omegaconf import OmegaConf, Container
-                except ModuleNotFoundError:
-                    pass
-                else:
-                    # Convert to primitives
-                    if isinstance(params, Container):
-                        params = OmegaConf.to_container(params, resolve=True)
-
-                hparams = self.experiment.meta_run_attrs_tree.subtree('hparams')
-                for key, value in params.items():
-                    hparams.set(key, value, strict=False)
-
-            @rank_zero_only
-            def log_metrics(self, metrics: Dict[str, float],
-                            step: Optional[int] = None):
-                assert rank_zero_only.rank == 0, \
-                    'experiment tried to log from global_rank != 0'
-
-                for k, v in metrics.items():
-                    name = k
-                    context = {}
-                    if self._train_metric_prefix \
-                            and name.startswith(self._train_metric_prefix):
-                        name = name[len(self._train_metric_prefix):]
-                        context['subset'] = 'train'
-                    elif self._test_metric_prefix \
-                            and name.startswith(self._test_metric_prefix):
-                        name = name[len(self._test_metric_prefix):]
-                        context['subset'] = 'test'
-                    elif self._val_metric_prefix \
-                            and name.startswith(self._val_metric_prefix):
-                        name = name[len(self._val_metric_prefix):]
-                        context['subset'] = 'val'
-                    self.experiment.track(v, name=name, context=context)
-
-            @rank_zero_only
-            def close(self) -> None:
-                super().close()
-
-            @property
-            def save_dir(self) -> str:
-                return self.experiment.repo.path
-
-            @property
-            def name(self) -> str:
-                return self._experiment_name
-
-            @property
-            def version(self) -> str:
-                return self.experiment.hash
-
-        cls.__pl_logger_cls = _PytorchLightningLogger
-        return cls.__pl_logger_cls
-
+class AimLogger(LightningLoggerBase):
     def __init__(self,
                  repo: Optional[str] = None,
                  experiment: Optional[str] = None,
                  train_metric_prefix: Optional[str] = 'train_',
                  val_metric_prefix: Optional[str] = 'val_',
                  test_metric_prefix: Optional[str] = 'test_',
                  system_tracking_interval: Optional[int]
-                 = DEFAULT_SYSTEM_TRACKING_INT,
-                 ):
-        pass
+                 = DEFAULT_SYSTEM_TRACKING_INT,):
```

Comment:
worth adding system params auto-tracking flag as well. check the `Run.__init__` for details.

Reply:
Done ✅